lib/capybara_select2/helpers.rb

```diff
@@ -10,9 +10,9 @@ def select2(*args)
       Utils.validate_options!(options)
 
       container = if options[:xpath]
-        find(:xpath, options[:xpath])
+        find(:xpath, options[:xpath], visible: :all)
```

Comment:
also this was needed, since now the `<select>` tag is not visibile, but we still want to find it and then use the `.sibling()` method to find the actual `select2-container`